variables.tf

```diff
@@ -18,6 +18,28 @@ variable "asset_dir" {
   type        = "string"
 }
 
+variable "cluster_id" {
+  description = "The identifier for the cluster for AWS IAM authentication purposes"
+  type        = "string"
+}
+
+variable "admin_role_arns" {
+  description = "A list of ARNs that will be mapped to cluster administrators"
+  type        = "list"
+}
+
+variable "admin_role_arn_mapping_template" {
+  description = "The template that renders into yaml for the aws iam authenticator. Whitespace is important here."
+  type        = "string"
+
+  default = <<TEMPLATE
```

Comment:
This feels quite filthy. If someone can some up with a better way of doing this...have at it!

Reply:
Could we use a local, possibly with a template?

Reply:
As discussed, this would add another level of indirection to an already unnecessarily convoluted process and so is not desirable. Having it inline like this is better than splitting out into a template file.